server/src/application/application.controller.ts

```diff
@@ -1,8 +1,18 @@
-import { Controller } from '@nestjs/common';
-// import { Application } from './bcaap-application.entity';
+import { Controller, Get, Param, Post } from '@nestjs/common';
+import { Application } from './application.entity';
 import { ApplicationService } from './application.service';
 
 @Controller('applications')
 export class ApplicationController {
   constructor(private applicationService: ApplicationService) {}
+
+  @Get('/:applicationId')
+  getCurrentProfileSelection(@Param('applicationId') applicationId: string) {
```

Comment:
We'll also get the logged-in user data and fetch the logged-in user's applicationId. By this way, other users cannot view each other's applications.

Mark this as TODO

Reply:
Thank you, super good point! Marked it in a comment in the same function.

Will transfer it into a ticket once we get there.